unpackaged/tests/serviceDeliveryModal/lwc/testModal/testModal.js

```diff
@@ -1,7 +1,6 @@
 import { LightningElement, wire, track } from "lwc";
 import { CurrentPageReference } from "lightning/navigation";
-import { fireEvent } from "c/pubsub";
-import { debug, debouncify } from "c/util";
+import { debug } from "c/util";
```

Comment:
@AndersonTarren - do we still need this?

Reply:
I believe this component was initially created to use in testing the CaseMan modal directly from PMM.  If we don't feel like it's being used in that manner, or we might not use it in the future, I could see getting rid of it.